src/odemis/gui/comp/overlay/stage_point_select.py

```diff
@@ -29,27 +29,13 @@ class StagePointSelectOverlay(WorldOverlay):
     """ Overlay for moving the stage (in physical coordinates) upon the selection of canvas points"""
 
     def on_dbl_click(self, evt):
-        if self.active:
+        if self.active.value:
             v_pos = evt.Position
             p_pos = self.cnvs.view_to_phys(v_pos, self.cnvs.get_half_buffer_size())
             # directly move the stage to the selected physical position
             self.cnvs.view.moveStageTo(p_pos)
         else:
             WorldOverlay.on_dbl_click(self, evt)
 
-    def on_left_down(self, evt):
```

Comment:
@K4rishma this class is the base class for cryo feature overlay. Do you know what will happen if `on_left_down` and `on_left_up` are removed?

Reply:
I followed the software release procedure for METEOR and it seems to work fine with this change